### PR TITLE
[improve][ms] Bvars add the FDB get_count_normalized indicator

### PR DESCRIPTION
**Pull request by Sjwhello**
### What problem does this PR solve?
Bvars add the FDB get_count_normalized indicator
- g_bvar_txn_kv_ get_count_normalized

### CODE REVIEW

**Comment by Sjwhello** [comment]
run buildall

File: cloud/src/common/bvars.cpp [hunk 1]
```diff
@@ -102,6 +102,8 @@ bvar::LatencyRecorder g_bvar_txn_kv_get_read_version("txn_kv", "get_read_version
 bvar::LatencyRecorder g_bvar_txn_kv_get_committed_version("txn_kv", "get_committed_version");
 bvar::LatencyRecorder g_bvar_txn_kv_batch_get("txn_kv", "batch_get");
 
+bvar::Adder<int64_t> g_bvar_txn_kv_get_count_total("txn_kv", "get_count_total");
```
Comment (gavinchou) [inline cloud/src/common/bvars.cpp:105]: suggest name `get_count_normalized`

Comment (Sjwhello) [reply cloud/src/common/bvars.cpp]: done

**Comment by Sjwhello** [comment]
run buildall

**Comment by Sjwhello** [comment]
run buildall

**Comment by Sjwhello** [comment]
run cloud_p0

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**